colorizer_data/types.py

```diff
@@ -139,20 +139,22 @@ class ColorizerMetadata(DataClassJsonMixin):
     author: Optional[str] = None
     dataset_version: Optional[str] = None
     """User-defined dataset version."""
-    date_created: Optional[str] = None
+
+    # Internal use
+    _date_created: Optional[str] = None
     """ISO-formatted datetime string in UTC. See `DATETIME_FORMAT`."""
-    last_modified: Optional[str] = None
+    _last_modified: Optional[str] = None
```

Comment:
There is one interesting twist on this - the scientists might want something they can display that isn't LITERALLY the last time the file was modified but more like a "last revision date of our dataset" which could be tied to a release or paper or something.  TBH, date created and last modified are filesystem-level attributes anyway.  So maybe these could be not necessarily internal?

Reply:
That makes sense to me, I can revert this change for the date-related fields